src/lib/browse/layers/lines/Gradients.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
+  import OsmLicense from "../OsmLicense.svelte";
+  import OsOglLicense from "../OsOglLicense.svelte";
+  import {
+    ExternalLink,
+    HelpButton,
+    Popup,
+    publicResourceBaseUrl,
+  } from "lib/common";
+  import { Checkbox } from "lib/govuk";
+  import { layerId, makeColorRamp } from "lib/maplibre";
+  import {
+    hoverStateFilter,
+    LineLayer,
+    VectorTileSource,
+    SymbolLayer,
+  } from "svelte-maplibre";
+  import { colors, denseLineWidth } from "../../colors";
+  import SequentialLegend from "../SequentialLegend.svelte";
+
+  let name = "gradient";
+  let colorScale = colors.gradient_flat_to_steep;
+  let limits = [0, 3, 5, 8, 10, 20, 100];
```

Comment:
These limits are a bit strange compared to the legend:

For example if you have a gradient of 10.1 it shows you the colour associated with 20 on the legend.
I wonder if we want...


![image](https://github.com/acteng/atip/assets/133757676/f8b92dcc-7754-4e71-b4ad-1632277e5957)
Looking at the legend again, 0 - 3 look right: the numbers represent the breakpoints and the legend reflects this. By the time we reach 8 and 10 it looks like the numbers are at the centre of that colour, as if the band of yellow should exist centred on 8, and the band of bright red should be centred on 10. Not sure the best solution here.

Reply:
To explain what I mean: looking at that legend I'd expect 15 to be where we switch from bright to slightly dark red

Reply:
Agreed, this is confusing. https://github.com/acteng/atip/blob/main/src/lib/browse/layers/SequentialLegend.svelte is drawing the legend and just trying to evenly distribute whitespace. But really we want to line up exactly. Any ideas how to do this better?